src/main/scala/io/buoyant/linkerd/RateLimiterConfig.scala

```diff
@@ -0,0 +1,26 @@
+package io.buoyant.linkerd
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.finagle.http.{Request, Response}
+import com.twitter.finagle.util.DefaultTimer
+import com.twitter.finagle.{Filter, Stack}
+import io.buoyant.linkerd.protocol.HttpRequestAuthorizerConfig
+
+class RateLimiterConfig(limit: Int, intervalSeconds: Int = 1) extends HttpRequestAuthorizerConfig{
```

Comment:
elsewhere in Linkerd config we abbreviate parameter names like `intervalSecs` so it might be nice to be consistent with that